sources/packages/backend/apps/api/src/auth/decorators/institution/institution-bc-public.decorator.ts

```diff
@@ -0,0 +1,12 @@
+import { SetMetadata } from "@nestjs/common";
+
+/**
+ * Allow only BC public institution to access the API.
+ */
+export const INSTITUTION_IS_BC_PUBLIC_KEY = "institution-is-bc-public-key";
+/**
+ * Decorator to provide context that it's consumer must be
```

Comment:
Just a suggestion, we can avoid starting the comment stating what the commented target is, for instance, "Method for", "Class for",  "Endpoint for", etc.